app/forms/teacher_interface/qualification_form.rb

```diff
@@ -55,6 +55,15 @@ def update_model
 
     delegate :application_form, to: :qualification
 
+    def qualifications_information
+      region = qualification.application_form.region
+
+      (
+        region.qualifications_information.presence ||
+          region.country.qualifications_information
+      )
+    end
+
```

Comment:
I'm not sure about this going in the form, generally we've tried to keep the forms only responsible for saving to the database and validation. We have started introducing "view objects" which might be a better place to put this, although for something as small as this I'd be tempted to put it directly in to the view.

Reply:
Thanks @thomasleese I introduced a view object as this was easier to unit test variations. Wasn't 100% sure it should be `QualificationViewObject` but that seems to fit with the controller & action.